scripts/image-generation.js

```diff
@@ -90,7 +90,7 @@ const generateTeraSaturdayImage = (
     ctx.fillText(postid, pst + c.width / 2 + 60, c.height / 3 + 120);
     ctx.restore();
 
-    const buffer = canvas.toBuffer("image/png");
+    const buffer = canvas.toBuffer("image/jpg");
```

Comment:
Wait, we created a PNG buffer and then wrote it out with a `.jpg` file extension? :zany_face: